### DISCUSSION TITLE
Manual/dynamic-batching

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#642] [Notes]
# Manual/dynamic-batching

https://docs.unity3d.com/Manual/dynamic-batching.html

**Comment by radiatoryang** [comment]
Currently in URP, shader graph sprite shaders seem to ALWAYS have dynamic batching forced on, even if you uncheck Dynamic Batching. This can cause problems if you're doing any object space vertex deformation in the sprite shader. 

The only way to disable dynamic batching for sprites is to set the ShaderLab tag DisableBatching = true ... but Shader Graph doesn't support adding SL tags, so you'll have to copy and paste the generated shader code and add the DisableBatching tag yourself. See https://forum.unity.com/threads/how-to-prevent-sprite-batching-or-display-sprites-without-using-sprite-renderer.954894/ and https://docs.unity3d.com/Manual/SL-SubShaderTags.html
